README.md

```diff
@@ -54,7 +54,7 @@ See below code sample (`examples/simple.go`) for a usage example. For detailed u
         PaddingA:    "WillBePadded",
     }
 
-    doctree, _ := proofs.NewDocumentTree(proofs.TreeOptions{Hash: sha256.New()})
+    doctree, _ := proofs.NewDocumentTree(proofs.TreeOptions{Hash: sha256.New(), LeafHash: md5.New()})
```

Comment:
I'd rather not use md5 in the example since it's broken. How about using blake2b?

Reply:
updated